spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryDirectDataSourceWriterContext.java

```diff
@@ -50,7 +50,7 @@
   private final String writeUUID;
   private final RetrySettings bigqueryDataWriterHelperRetrySettings;
 
-  private final TableId temporaryTableId;
+  private final BigQueryTable writtenTable;
```

Comment:
Minor: Can we change the name to something like `tableToWrite`? Since it is possible that the table does not exist and `writtenTable` implies that we have already written something to it